### PR TITLE
[Source-postgres] Set default cursor value for cdc mode

### PR DESCRIPTION
**Pull request by nguyenaiden**
## What
[26807](https://app.zenhub.com/workspaces/db--dw-source-connectors-6333360e0a41155061efbcbd/issues/gh/airbytehq/airbyte/26807)
To prepare for Destination v2, Cdc-enabled DB sources need to have a pre-defined cursor for normalization to operate properly. 
## How
During the discover phase, `source_defined_cursor` is set to `true`, and `_ab_cdc_lsn` is selected as the predefined cursor. NOTE: This value could change if we believe it's not unique. 
image

`AirbyteStateMessage:`
```
{
  "cdc": true,
  "cdc_state": {
    "state": {
      "[\"postgres\",{\"server\":\"postgres\"}]": "{\"transaction_id\":null,\"lsn\":840386496,\"txId\":1895971,\"ts_usec\":1686867930174234}"
    }
  },
  "streams": [
    {
      "stream_name": "blah",
      "stream_namespace": "public",
      "cursor_field": [
        "_ab_cdc_lsn"
      ]
    },
    {
      "stream_name": "jsonb_test_norm",
      "stream_namespace": "public",
      "cursor_field": [
        "_ab_cdc_lsn"
      ]
    }
  ]
}
```

## Recommended reading order
1. `PostgresCatalogHelper.java`
2. `PostgresSource.java`
3.  All the test files

## 🚨 User Impact 🚨
New CDC syncs that have refreshed their source schema will see this cursor field be chosen. Customers who reset their data will continue to see old behavior until their source goes through the `discover()` phase again.

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-alloydb-strict-encrypt test report (commit [`fa057b48c9`](https://github.com/airbytehq/airbyte/pull/27442/commits/fa057b48c92f9f4d07baad79df80a06c9468e5aa)) - ❌

⏲️  Total pipeline duration: 369 seconds

| Step                                                                                 | Result   |
|:-------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-alloydb-strict-encrypt/metadata.yaml | ✅        |
| Connector version semver check.                                                      | ✅        |
| Connector version increment check.                                                   | ❌        |
| QA checks                                                                            | ✅        |
| Build connector tar                                                                  | ✅        |
| Build source-alloydb-strict-encrypt docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                           | ✅        |
| Integration tests                                                                    | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5293066256)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-alloydb-strict-encrypt test
```

**Comment by nguyenaiden** [comment]
/legacy-test connector=connectors/source-mysql-strict-encrypt

**Comment by nguyenaiden** [comment]
/legacy-test connector=connectors/source-mssql-strict-encrypt

**Comment by octavia-squidington-iii** [comment]
##  source-postgres-strict-encrypt test report (commit [`fa057b48c9`](https://github.com/airbytehq/airbyte/pull/27442/commits/fa057b48c92f9f4d07baad79df80a06c9468e5aa)) - ❌

⏲️  Total pipeline duration: 610 seconds

| Step                                                                                  | Result   |
|:--------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-postgres-strict-encrypt/metadata.yaml | ✅        |
| Connector version semver check.                                                       | ✅        |
| Connector version increment check.                                                    | ❌        |
| QA checks                                                                             | ✅        |
| Build connector tar                                                                   | ✅        |
| Build source-postgres-strict-encrypt docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                            | ✅        |
| Integration tests                                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5293066256)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  source-mssql test report (commit [`fa057b48c9`](https://github.com/airbytehq/airbyte/pull/27442/commits/fa057b48c92f9f4d07baad79df80a06c9468e5aa)) - ❌

⏲️  Total pipeline duration: 2163 seconds

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mssql/metadata.yaml | ✅        |
| Connector version semver check.                                     | ✅        |
| Connector version increment check.                                  | ❌        |
| QA checks                                                           | ✅        |
| Build connector tar                                                 | ✅        |
| Build source-mssql docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                          | ✅        |
| Integration tests                                                   | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5293066256)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mssql test
```

**Comment by octavia-squidington-iii** [comment]
##  source-alloydb test report (commit [`fa057b48c9`](https://github.com/airbytehq/airbyte/pull/27442/commits/fa057b48c92f9f4d07baad79df80a06c9468e5aa)) - ❌

⏲️  Total pipeline duration: 321 seconds

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-alloydb/metadata.yaml | ✅        |
| Connector version semver check.                                       | ✅        |
| Connector version increment check.                                    | ❌        |
| QA checks                                                             | ✅        |
| Build connector tar                                                   | ✅        |
| Build source-alloydb docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Integration tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5293066256)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-alloydb test
```

**Comment by octavia-squidington-iii** [comment]
##  source-alloydb-strict-encrypt test report (commit [`48b836dd39`](https://github.com/airbytehq/airbyte/pull/27442/commits/48b836dd39c969854d490e8281aedf07dbe2c0c9)) - ❌

⏲️  Total pipeline duration: 325 seconds

| Step                                                                                 | Result   |
|:-------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-alloydb-strict-encrypt/metadata.yaml | ✅        |
| Connector version semver check.                                                      | ✅        |
| Connector version increment check.                                                   | ❌        |
| QA checks                                                                            | ✅        |
| Build connector tar                                                                  | ✅        |
| Build source-alloydb-strict-encrypt docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                           | ✅        |
| Integration tests                                                                    | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5293067095)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-alloydb-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  source-mssql test report (commit [`48b836dd39`](https://github.com/airbytehq/airbyte/pull/27442/commits/48b836dd39c969854d490e8281aedf07dbe2c0c9)) - ❌

⏲️  Total pipeline duration: 2162 seconds

| Step                                                                | Result   |
|:--------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mssql/metadata.yaml | ✅        |
| Connector version semver check.                                     | ❌        |
| Connector version increment check.                                  | ❌        |
| QA checks                                                           | ✅        |
| Build connector tar                                                 | ✅        |
| Build source-mssql docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                          | ✅        |
| Integration tests                                                   | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5293067095)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mssql test
```

**Comment by octavia-squidington-iii** [comment]
##  source-mysql-strict-encrypt test report (commit [`48b836dd39`](https://github.com/airbytehq/airbyte/pull/27442/commits/48b836dd39c969854d490e8281aedf07dbe2c0c9)) - ❌

⏲️  Total pipeline duration: 1796 seconds

| Step                                                                               | Result   |
|:-----------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mysql-strict-encrypt/metadata.yaml | ✅        |
| Connector version semver check.                                                    | ✅        |
| Connector version increment check.                                                 | ❌        |
| QA checks                                                                          | ✅        |
| Build connector tar                                                                | ✅        |
| Build source-mysql-strict-encrypt docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                         | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5293067095)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  source-postgres test report (commit [`48b836dd39`](https://github.com/airbytehq/airbyte/pull/27442/commits/48b836dd39c969854d490e8281aedf07dbe2c0c9)) - ❌

⏲️  Total pipeline duration: 1247 seconds

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-postgres/metadata.yaml | ✅        |
| Connector version semver check.                                        | ✅        |
| Connector version increment check.                                     | ❌        |
| QA checks                                                              | ✅        |
| Build connector tar                                                    | ✅        |
| Build source-postgres docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                             | ✅        |
| Integration tests                                                      | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5293067095)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres test
```

**Review by prateekmukhedkar — COMMENTED** [review]
So just to confirm, `_ab_cdc_lsn` field and associated value is already present in the data that we serialize, and that is why we don't need to add this field as we reading data?

**Comment by nguyenaiden** [comment]
@prateekmukhedkar yes, we are currently emitting the lsn value in the data we serialize so we don't need to add a new column.

**Comment by edgao** [comment]
running legacy normalization tests in https://github.com/airbytehq/airbyte/pull/27670#issuecomment-1604637535. Will approve once those look good.

**Review by edgao — APPROVED** [review]
tests are green on https://github.com/airbytehq/airbyte/pull/27670. :shipit: 

thanks for putting this together so quickly!

**Review by subodh1810 — COMMENTED** [review]
THis looks fine. I just want you to carry out few tests
1. Create a postgres source cdc connector on an old version with a destination that supports normalization (snowflake), run a sync and then change image version of the connector to use changes from this PR and then run a sync and make sure everything works.
2. Do everything as mentioned in step 1 but this time refresh the schema as well.
3. Create a connector on this new version and then roll back to the old version and see what happens (both with and without refresh schema)

**Comment by octavia-squidington-iii** [comment]
##  source-postgres test report (commit [`052a5aee9d`](https://github.com/airbytehq/airbyte/pull/27442/commits/052a5aee9d1687bcff35ffd5ddceb95ebb5286ea)) - ❌

⏲️  Total pipeline duration: 10mn18s 

| Step                                                                       | Result   |
|:---------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-postgres/metadata.yaml     | ✅        |
| Connector version semver check                                             | ✅        |
| QA checks                                                                  | ✅        |
| Build connector tar                                                        | ✅        |
| Build source-postgres docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-postgres:integrationTest | ✅        |
| Acceptance tests                                                           | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/duy_cdc-source-defined-cursor/1689185662/052a5aee9d1687bcff35ffd5ddceb95ebb5286ea/source-postgres/2.1.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres test
```

**Comment by octavia-squidington-iii** [comment]
##  source-mysql test report (commit [`052a5aee9d`](https://github.com/airbytehq/airbyte/pull/27442/commits/052a5aee9d1687bcff35ffd5ddceb95ebb5286ea)) - ❌

⏲️  Total pipeline duration: 01mn29s 

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mysql/metadata.yaml     | ✅        |
| Connector version semver check                                          | ✅        |
| QA checks                                                               | ✅        |
| Build connector tar                                                     | ✅        |
| Build source-mysql docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-mysql:integrationTest | ❌        |
| Acceptance tests                                                        | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/duy_cdc-source-defined-cursor/1689185662/052a5aee9d1687bcff35ffd5ddceb95ebb5286ea/source-mysql/2.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql test
```

**Comment by octavia-squidington-iii** [comment]
##  source-postgres-strict-encrypt test report (commit [`052a5aee9d`](https://github.com/airbytehq/airbyte/pull/27442/commits/052a5aee9d1687bcff35ffd5ddceb95ebb5286ea)) - ❌

⏲️  Total pipeline duration: 05mn43s 

| Step                                                                                      | Result   |
|:------------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-postgres-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                            | ✅        |
| QA checks                                                                                 | ✅        |
| Build connector tar                                                                       | ✅        |
| Build source-postgres-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-postgres-strict-encrypt:integrationTest | ✅        |
| Acceptance tests                                                                          | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/duy_cdc-source-defined-cursor/1689185662/052a5aee9d1687bcff35ffd5ddceb95ebb5286ea/source-postgres-strict-encrypt/2.1.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-postgres-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  source-mysql-strict-encrypt test report (commit [`052a5aee9d`](https://github.com/airbytehq/airbyte/pull/27442/commits/052a5aee9d1687bcff35ffd5ddceb95ebb5286ea)) - ❌

⏲️  Total pipeline duration: 12mn30s 

| Step                                                                                   | Result   |
|:---------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mysql-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                         | ✅        |
| QA checks                                                                              | ✅        |
| Build connector tar                                                                    | ✅        |
| Build source-mysql-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-mysql-strict-encrypt:integrationTest | ❌        |
| Acceptance tests                                                                       | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/duy_cdc-source-defined-cursor/1689185662/052a5aee9d1687bcff35ffd5ddceb95ebb5286ea/source-mysql-strict-encrypt/2.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  source-alloydb-strict-encrypt test report (commit [`052a5aee9d`](https://github.com/airbytehq/airbyte/pull/27442/commits/052a5aee9d1687bcff35ffd5ddceb95ebb5286ea)) - ❌

⏲️  Total pipeline duration: 04mn18s 

| Step                                                                                     | Result   |
|:-----------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-alloydb-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                           | ✅        |
| QA checks                                                                                | ✅        |
| Build connector tar                                                                      | ✅        |
| Build source-alloydb-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-alloydb-strict-encrypt:integrationTest | ✅        |
| Acceptance tests                                                                         | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/duy_cdc-source-defined-cursor/1689185662/052a5aee9d1687bcff35ffd5ddceb95ebb5286ea/source-alloydb-strict-encrypt/2.0.28/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-alloydb-strict-encrypt test
```

**Comment by nguyenaiden** [comment]
/legacy-test connector=connectors/source-postgres-strict-encrypt

File: airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java [hunk 1]
```diff
@@ -275,6 +275,7 @@ public AirbyteCatalog discover(final JsonNode config) throws Exception {
           .map(PostgresCatalogHelper::overrideSyncModes)
           .map(PostgresCatalogHelper::removeIncrementalWithoutPk)
           .map(PostgresCatalogHelper::setIncrementalToSourceDefined)
+          .map(PostgresCatalogHelper::setDefaultCursorFieldForCdc)
```
Comment (pedroslopez) [inline airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java:278]: 👍🏼 only for cdc, we are setting the default cursor.

**Comment by nguyenaiden** [comment]
/legacy-test connector=connectors/source-alloydb-strict-encrypt

**Review by pedroslopez — APPROVED** [review]
Didn't review much on the tests side, leaving that to the sources team. 

From ops side looks good, seeing breaking changes checks correctly indicated as skipped in the test logs for the affected version:

```
SKIPPED [1] ../usr/local/lib/python3.9/site-packages/connector_acceptance_test/tests/test_core.py:624: Backward compatibility tests are disabled for version 2.1.1.
```

Should make sure acceptance tests are passing for the connectors to be published - it looks like they are.

Build failure for destination-snowflake is also failing on master and this PR does not touch it, so should be ok to release.

**Comment by octavia-approvington** [comment]
Our work here is done <br/> done

**Review by octavia-approvington — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-mysql-strict-encrypt test report (commit [`5c2e4796ab`](https://github.com/airbytehq/airbyte/pull/27442/commits/5c2e4796aba13f1e92410e2b5a5ee24679323e48)) - ❌

⏲️  Total pipeline duration: 12mn58s 

| Step                                                                                   | Result   |
|:---------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mysql-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                         | ✅        |
| QA checks                                                                              | ✅        |
| Build connector tar                                                                    | ✅        |
| Build source-mysql-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-mysql-strict-encrypt:integrationTest | ❌        |
| Acceptance tests                                                                       | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/duy_cdc-source-defined-cursor/1689275092/5c2e4796aba13f1e92410e2b5a5ee24679323e48/source-mysql-strict-encrypt/2.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mysql-strict-encrypt test
```

**Pull request merged**